contributing-docs/02_how_to_communicate.rst

```diff
@@ -0,0 +1,151 @@
+    under the License.
+
+How to communicate
+==================
+
+Apache Airflow is a Community within Apache Software Foundation. As the motto of
+the Apache Software Foundation states "Community over Code" - people in the
+community are far more important than their contribution.
+
+This means that communication plays a big role in it, and this chapter is all about it.
+
+In our communication, everyone is expected to follow the `ASF Code of Conduct <https://www.apache.org/foundation/policies/conduct>`_.
+
+.. contents:: :local:
+
+Various Communication channels
+------------------------------
+
+We have various channels of communication - starting from the official devlist, comments
+in the PR, Slack, wiki.
+
+All those channels can be used for different purposes.
+You can join the channels via links at the `Airflow Community page <https://airflow.apache.org/community/>`_
+
+* The `Apache Airflow devlist <https://lists.apache.org/list.html?dev@airflow.apache.org>`_ for:
+   * official communication
+   * general issues, asking community for opinion
+   * discussing proposals
+   * voting
+* The `Airflow CWiki <https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Home?src=breadcrumbs>`_ for:
+   * detailed discussions on big proposals (Airflow Improvement Proposals also name AIPs)
+   * helpful, shared resources (for example Apache Airflow logos
+   * information that can be reused by others (for example instructions on preparing workshops)
+* GitHub `Pull Requests (PRs) <https://github.com/apache/airflow/pulls>`_ for:
+   * discussing implementation details of PRs
+   * not for architectural discussions (use the devlist for that)
+* The deprecated `JIRA issues <https://issues.apache.org/jira/projects/AIRFLOW/issues/AIRFLOW-4470?filter=allopenissues>`_ for:
+   * checking out old but still valuable issues that are not on GitHub yet
+   * mentioning the JIRA issue number in the title of the related PR you would like to open on GitHub
+
+**IMPORTANT**
+We don't create new issues on JIRA anymore. The reason we still look at JIRA issues is that there are valuable
+tickets inside of it. However, each new PR should be created on `GitHub issues <https://github.com/apache/airflow/issues>`_
+as stated in `Contribution Workflow Example <contribution-workflow.rst>`_
+
+Slack details
+-------------
+
+* The `Apache Airflow Slack <https://s.apache.org/airflow-slack>`_ for:
+   * ad-hoc questions related to development (#development channel)
+   * asking for review (#development channel)
+   * asking for help with first contribution PRs (#development-first-pr-support channel)
+   * troubleshooting (#troubleshooting channel)
+   * group talks (including SIG - special interest groups) (#sig-* channels)
+   * notifications (#announcements channel)
+   * random queries (#random channel)
+   * regional announcements (#users-* channels)
+   * occasional discussions (wherever appropriate including group and 1-1 discussions)
```

Comment:
We can also make mention of the breeze and the airflow-meetups channel

Reply:
Good point.